README.md

```diff
@@ -1 +1,74 @@
-# Weather-Now
\ No newline at end of file
+# Your daily weather
+
+> This app enables users to get instant Information regarding weather conditions. If you share your location, it will fetch the top 10 cities near you and all the weather detail for each.
+
+> you can also search for any city in the world and get the detail. It makes use of 3 different APIs. To fetch data, map coordinates, and locate your area and cities near you.
+
+## Built With
+
+- JavaScript
+- React
+- Redux
+- API Gateway
+
+## Screenshots
+
+### Mobile
+
+<img src="./src/resources/weather-mobile.PNG" width="45%"></img>
+<img src="./src/resources/weather-mobile2.PNG" width="45%"></img>
+
+### Desktop
+
+<img src="./src/resources/weather-list.PNG" width="100%"></img>
+
+## Live Demo
```

Comment:
- [ ] In live link the `search` for cities does not work, throws an error in console, kindly update it

Reply:
### Hello @GzimAsani, Thank you for the review. I think the problem with that is actually `netlify`. If you start it with the local server it will work just fine. But I sure will work on it.

Reply:
- Yes locally was working fine, I thought you forgot to deploy the latest version of your project

Reply:
@GzimAsani Thanks for that, nad no I did not forget. 
I have checked it, I have tried to fix it for a while. but `netlify` have some rules blocking the API provider.
![error](https://user-images.githubusercontent.com/69077061/169145080-c49c3102-9f7a-4f96-aed1-6dc4bd56805d.PNG).
🙏🏼